keras/backend/jax/nn.py

```diff
@@ -546,7 +546,7 @@ def batch_normalization(
         offset = jnp.reshape(offset, shape)
         res = res + offset
 
-    return x * inv + res
+    return jnp.add(x + inv, res)
```

Comment:
That should be `*`

Reply:
Thanks for catching, updated accordingly.